examples/bench.ts

```diff
@@ -1,7 +1,7 @@
 import {
   Tensor,
   rand
-} from "shumaiml";
+} from "../shumai";
```

Comment:
needs to be "shumaiml" again :)

Reply:
Oops, forgot to revert after testing; thanks!